features/configuration/fail_if_no_examples.feature

```diff
@@ -0,0 +1,31 @@
+Feature: fail if no examples
+
+  Use the `fail_if_no_examples` option to make RSpec exit with a failure status (by default 1) if there are 0 examples. Using this option, it is recommended to add a `--require spec_helper` option to `.rspec` file to ensure the `fail_if_no_examples` option is set even if no spec files are loaded.
```

Comment:
Suggestion: s/if there are 0 examples/if there are no examples/.

I think it reads more naturally and aligns with the config option name.